### PR TITLE
Implemented DELETE /api/v1/observer endpoint in observers list.

### LINKED ISSUE #293: [Observers] Call delete endpoint from observers list
The delete observer action is available in the observers lists, in the action dropdown. When pressed it should show a confirmation dialog and if the user confirms the action, the delete should be called `DELETE ​/api​/v1​/observer`

This is very similar to the delete functionality from the edit observer page. Please check that out.

### PR DESCRIPTION
**Pull request by VladCuciureanu**
### What does it fix?

Closes #293

Implements the logic behind the Delete button in the observer list item dropdown.
Uses 'DELETE ​/api​/v1​/observer' endpoint.

### How has it been tested?

Tested manually using local environment configs.

### CODE REVIEW

File: src/app/components/observers/observers.component.ts [hunk 1]
```diff
@@ -263,6 +259,17 @@ export class ObserversComponent implements OnInit, OnDestroy {
     this.observersCountSubscription.unsubscribe();
   }
 
+  deleteObserver(observer: Observer) {
+    if (confirm('Are you sure you want to remove this observer?')) {
```
Comment (idormenco) [inline src/app/components/observers/observers.component.ts:263]: can we use translation service for this message (also other bellow )

File: src/app/components/observers/observers.component.ts [hunk 2]
```diff
@@ -263,6 +259,17 @@ export class ObserversComponent implements OnInit, OnDestroy {
...
+      this.observersService
+        .deleteObserver(observer.id)
+        .subscribe(() => {
+          this.toastrService.warning('Success', 'Observer has been removed');
```
Comment (idormenco) [inline src/app/components/observers/observers.component.ts:267]: this one too

**Review by idormenco — CHANGES_REQUESTED** [review]

**Review by idormenco — APPROVED** [review]
Looks good to me

**Comment by VladCuciureanu** [comment]
> Looks good to me

Sorry! I added the localized strings to a previous implementation of the same dialog. You might have to approve the pull request again 😅

**Review by idormenco — APPROVED** [review]

**Review by aniri — APPROVED** [review]

**Pull request merged**